README.md

```diff
@@ -33,3 +33,45 @@ If you wish to setup some other host, you will need to edit your inventory. Repl
     ansible-playbook -i inventory site.yml
 
 Navigate to ~/architecture2 and follow the rest of the pancancer-info instructions on how to setup and use bindle
+
+## Upgrading to a new version
+
+If you have a launcher that has an older version of architecture-setup and you wish to upgrade, you can use the `upgrade_architecture_setup.sh` script.
+
+This script will get the file `roles/bindle-profiles/vars/main.yml` that is attached to the latest released version of architecture-setup. It is also possible to specify a specific version of architecture-setup to upgrade to. The script will then run the main ansible playbook for architecture-setup, using the updated `roles/bindle-profiles/vars/main.yml`.
+
+Example:
+
+    upgrade_architecture_setup.sh -v 1.0.5
+    
+The file `roles/bindle-profiles/vars/main.yml` that is a part of architecture-setup 1.0.5 will be downloaded to the current local repository. Ansible will then be run (`sudo ansible-playbook -i inventory site.yml`).
```

Comment:
Attach the error message that shows up when you run without sudo, I'm curious what the issue is since individual ansible steps should automatically use sudo as needed.